extensions/udapdr/scripts/dsp_utilities/distill_triples_with_retriever.py

```diff
@@ -0,0 +1,25 @@
+
+from primeqa.ir.dense.colbert_top.colbert.infra.run import Run
+from primeqa.ir.dense.colbert_top.colbert.infra.config import ColBERTConfig, RunConfig
+from primeqa.ir.dense.colbert_top.colbert import Trainer
+
+def distill_triples_with_retriever(given_triples, given_synth_queries, given_checkpoint, chosen_split, chosen_type, chosen_set, LoTTE_or_BEIR, chosen_BEIR_set, chosen_BEIR_type):
+    
+    with Run().context(RunConfig(nranks=4)):
+        
+        if LoTTE_or_BEIR == "LoTTE":
+        	collection = '../downloads/lotte/' + chosen_split + '/' + chosen_set +'/collection.tsv'
```

Comment:
Can we make the root path configurable as a command line argument instead of hardcoding it to `../downloads`?

Reply:
Same with any other relative path